pkg/resourceinterpreter/interpreter.go

```diff
@@ -146,11 +146,6 @@ func (i *customResourceInterpreterImpl) Retain(desired *unstructured.Unstructure
 func (i *customResourceInterpreterImpl) AggregateStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
 	klog.V(4).Infof("Begin to aggregate status for object: %v %s/%s.", object.GroupVersionKind(), object.GetNamespace(), object.GetName())
 
-	// If status has not been collected, there is no need to aggregate.
-	if len(aggregatedStatusItems) == 0 {
```

Comment:
But I still think we should update status when the aggregated status is empty?

Reply:
good point!
@XiShanYongYe-Chang Please confirm.

Reply:
When a `job` resource is propagated, the status slice is empty for a short time when the status collection starts. In this case, the job status is set to `Completed`. We need to figure out a way to be compatible.

Reply:
> When a job resource is propagated, the status slice is empty for a short time when the status collection starts.

Could you please explain why? I'm asking because I want to ensure my fix is correct

Reply:
At the beginning of the status collection, the resource status remains empty for a short period of time.

Reply:
Yeah I think I get it, this happens when the binding is not scheduled yet, thanks